content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -1,130 +1,106 @@
 
-Marking a variable as sensitive prevents anybody (including you) from viewing its value in the variables section of the workspace in Terraform Cloud's UI or with its Variables API endpoint.
+If variables from `.auto.tfvars` files have the same key as existing variables in the Terraform Cloud workspace, the variables applied to the workspace overwrite variables from the files.
 
-Users with permission to read and write variables can set new values for sensitive variables.  No other attribute of a sensitive variable can be modified. To update other attributes, delete the variable and create a new variable to replace it.
+There are two additional cases when a workspace may have conflicting variables with the same key. You may use workspace-specific variables to [overwrite variables from variable sets](/docs/cloud/workspaces/managing-variables.html#overwrite-variable-sets) or there may be two variables from different variable sets with the same key.
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+Workspace-specific variables always overwrite variables from variable sets that have the same key.
 
-~> **Important:** Terraform runs will receive the full text of sensitive variables, and might print the value in logs and state files if the configuration pipes the value through to an output or a resource parameter. Additionally, Sentinel mocks downloaded from runs will contain the sensitive values of Terraform (but not environment) variables. Take care when writing your configurations to avoid unnecessary credential disclosure. Whenever possible, use environment variables since these cannot end up in state files or in Sentinel mocks. (Environment variables can end up in log files if TF_LOG is set to TRACE.)
+If different variable sets contain variables of the same type with the same key, Terraform Cloud prioritizes them first based on the variable set scope and then by the date that each variable set was applied to the workspace.
 
-### Looking Up Variable Names
+- **Scope:** Variable sets that apply to only a subset of workspaces take precedence over global variable sets that are applied to all workspaces within an organization.
 
-Terraform Cloud can't automatically discover variable names from a workspace's Terraform code. You must discover the necessary variable names by reading code or documentation, then enter them manually.
+- **Date Applied:** If non-global variable sets have conflicting variables, Terraform cloud prioritizes the variable set that was applied most recently. For example, if you apply Variable Set A and then apply Variable Set B to the same workspace a week later, Terraform Cloud will use any conflicting variables from Variable Set B. This will be the case regardless of which variable set has been edited most recently; Terraform Cloud only considers the date that each variable set is applied to the workspace when determining precedence.
```

Comment:
```suggestion
- **Date Applied:** If non-global variable sets have conflicting variables, Terraform Cloud uses values from the variable set that was applied most recently. For example, if you apply Variable Set A and then apply Variable Set B to the same workspace a week later, Terraform Cloud will use any conflicting variables from Variable Set B. This will be the case regardless of which variable set has been edited most recently; Terraform Cloud only considers the date that each variable set is applied to the workspace when determining precedence.
```